Gems/Atom/RHI/DX12/Code/Source/RHI/Device.cpp

```diff
@@ -29,6 +31,21 @@ namespace AZ
             void DeviceCompileMemoryStatisticsInternal(RHI::MemoryStatisticsBuilder& builder, IDXGIAdapterX* dxgiAdapter);
         }
 
+        Device::Device()
+        {
+            auto settingsRegistry = AZ::SettingsRegistry::Get();
+            RHI::Ptr<PlatformLimitsDescriptor> platformLimitsDescriptor = aznew PlatformLimitsDescriptor();
+            AZStd::string platformLimitsRegPath = AZStd::string::format("/Amazon/Atom/RHI/PlatformLimits/%s", RHI::Factory::Get().GetName().GetCStr());
```

Comment:
Lets move ""/Amazon/Atom/RHI/PlatformLimits" to a common file so it can be shared across all backends. Actually think about moving this function in the common layer if possible. It is currently duplicated across all backends.

Reply:
It looks like it's duplicated but in fact, PlatformLimitsDescriptor is backend dependent.
For this file, it's AZ::DX12::PlatformLimitsDescriptor, and will be casted to its super class AZ::RHI::PlatformLimitsDescriptor.
I still have to leave this part in those derived constructor. I can move the rest to their super class.